index.html

```diff
@@ -303,6 +281,45 @@
 	  </div>
 	</div>
 
+    <div id="research">
+      <div id="content" class="container">
+        <header>
+          <h2>Research directions</h2>
+        </header>
+        <!--
+        NOTE: for icon codes, see http://fontawesome.io/icons/
+        -->
+        <div class="row">
+          <section class="4u">
+              <!--
+              NOTE: fill this section next.
+              -->
+              &nbsp;
+          </section>
+          <section class="4u">
+            <span class="pennant"><span class="fa fa-connectdevelop"></span></span>
+              <h3>Machine Learning</h3>
+              <p>Making neural networks (NNs) performance-portable using <span class="lift-bold">Lift</span>: implementing generic and NN-specific optimizations as rewrite rules for efficient hardware utilization;
+        introducing traditional NN building blocks such as <i>conv</i>, <i>norm</i> and <i>fully_connected</i> for seemless integration with popular machine learning libraries.</p>

```

Comment:
for seamless integration of Lift with 
       ^^ typo here             ^^ add "of Lift"